Jenkinsfile

```diff
@@ -33,8 +33,8 @@ timestampedNode('java') {
     timeout(time: 180, unit: 'MINUTES') {
         // See below for what this method does - we're passing an arbitrary environment
         // variable to it so that JAVA_OPTS and MAVEN_OPTS are set correctly.
-        withMavenEnv(["JAVA_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m",
-                      "MAVEN_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m"]) {
+        withMavenEnv(["JAVA_OPTS=-Xmx2g -Xms512m -XX:MaxPermSize=1024m",
```

Comment:
OK I will be more specific in my review comment :-)

> Final Memory: 171M/**721M**

That is the maven build used a maximum of **721MB Heap** - so this tuning is OTT - and the original suffices and that is also way more than needed.